pkg/queue/poller.go

```diff
@@ -63,6 +63,7 @@ func (p *Poller) sync(stopCh <-chan struct{}) {
 	for {
 		select {
 		case listResultCh := <-p.listThreadCh:
+			time.Sleep(10 * time.Millisecond)
```

Comment:
need to find a way to go without this.

Reply:
Fixed in https://github.com/practo/k8s-worker-pod-autoscaler/pull/69/commits/81537618688edb07d3d0b0f33c031f6d896c598b